polynote-frontend/polynote/ui/component/notebook/notebookconfig.ts

```diff
@@ -401,15 +401,33 @@ class SparkConf {
 
 }
 
-class EnvConf {
+class KernelConf {
     readonly el: TagElement<"div">;
     private container: TagElement<"div">;
+    private jvmArgsInput: TagElement<"input">;
+    private scalaVersionInput: DropdownElement;
+
+    constructor(configState: StateView<NotebookConfig>) {
+        const envHandler = configState.view("env");
+        const jvmArgsHandler = configState.view("jvmArgs");
+        const scalaVersionHandler = configState.view("scalaVersion");
+
+        // TODO: this could come from the server
+        const availableScalaVersions = {
+            "2.11": "2.11",
+            "2.12": "2.12"
+        }
 
-    constructor(envHandler: StateView<Record<string, string> | undefined>) {
         this.el = div(['notebook-env', 'notebook-config-section'], [
-            h3([], ['Environment Variables']),
-            para([], ['Set environment variables here. Please note this is only supported when kernels are launched as a subprocess (default).']),
-            this.container = div(['env-list'], [])
+            h3([], ['Kernel configuration']),
+            para([], ['Please note this is only supported when kernels are launched as a subprocess (default).']),
```

Comment:
Consider changing this wording since it's no longer possible to launch kernels in-process.

Reply:
It is possible – if you're not using Spark and have `behavior.kernel_isolation = spark`.

Reply:
Oh, I guess I misunderstood the changes in the other PR. Thanks for the clarification.